moderator/moderate/views.py

```diff
@@ -124,9 +127,20 @@ def moderate_event(request, slug, q_id=None, accepted=None):
             question_form = QuestionForm(request.POST, instance=question)
             # update the question with moderator's reply
             if question_form.is_valid():
-                Question.objects.filter(id=question.pk).update(
-                    rejection_reason=question_form.cleaned_data.get("rejection_reason")
-                )
+                reason = question_form.cleaned_data.get("rejection_reason")
+                Question.objects.filter(id=question.pk).update(rejection_reason=reason)
+                if (
+                    reason
+                    and not question.is_accepted
+                    and question.submitter_contact_info
+                ):
+                    send_mail(
+                        SUBJECT,
+                        reason,
```

Comment:
Could we add some context to the email body, like:

> Your question for "Monthly internal meeting" has been rejected for the following reason:
>
> > This is the rejection reason
>
> If you'd like to amend and resubmit your question, please do so here: https://moderator.mozilla.org/e/monthly-internal-meeting

Reply:
Opened an issue to add an email template